### PR TITLE
Issue#849

### PR DESCRIPTION
**Pull request by reactexcel**

### CODE REVIEW

File: frontend/src/components/generic/AlertNotification.jsx [hunk 1]
```diff
@@ -12,11 +12,13 @@ const AlertNotification = ({message,handleRequestClose}) => {
   }
  
   return (
-    <Snackbar
+   <div className={nameOfClass}>
```
Comment (xmagestore) [inline frontend/src/components/generic/AlertNotification.jsx]: there is a seperate dsesign error/sucess message. don't need to use snackbar

File: frontend/src/modules/manageUsers/containers/manageUsers.jsx [hunk 2]
```diff
@@ -238,81 +238,85 @@ class ManageUsers extends React.Component {
     const { employee_life_cycle } = this.state;
     if (employee_life_cycle && Object.keys(employee_life_cycle).length) {
       let termination_error = false;
-      employee_life_cycle[5502].steps.map(step => {
-        termination_error = true;
+      employee_life_cycle[5502].steps.forEach(step => {
```
Comment (xmagestore) [inline frontend/src/modules/manageUsers/containers/manageUsers.jsx]: why is this 5502??

Comment (xmagestore) [reply frontend/src/modules/manageUsers/containers/manageUsers.jsx]: *what

File: frontend/src/components/generic/AlertNotification.jsx (hunk 1 above)
Comment (reactexcel) [reply frontend/src/components/generic/AlertNotification.jsx]: ok sir

File: frontend/src/modules/manageUsers/containers/manageUsers.jsx (hunk 2 above)
Comment (reactexcel) [reply frontend/src/modules/manageUsers/containers/manageUsers.jsx]: Sir It is object keys which is coming from backend.

**Pull request merged**